src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -2,46 +2,198 @@
 
 import java.util.List;
 
+@SuppressWarnings("unchecked")
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size = 0;
+    private Node<T> head;
+    private Node<T> tail;
+
+    class Node<T> {
```

Comment:
[Inner class should be at the end of the class.](https://github.com/mate-academy/jv-linked-list/blob/master/checklist.md#make-nodet-an-inner-class)

```suggestion
    private static class Node<T> {
```